src/Compilers/Test/Utilities/VisualBasic/TestOptions.vb

```diff
@@ -57,4 +57,15 @@ Friend Module TestOptionExtensions
     Public Function WithIOperationFeature(options As VisualBasicParseOptions) As VisualBasicParseOptions
         Return options.WithFeatures(options.Features.Concat(New KeyValuePair(Of String, String)() {New KeyValuePair(Of String, String)("IOperation", "true")}))
     End Function
+
+    <Extension()>
+    Public Function WithImplicitDefaultOptionalParameter(options As VisualBasicParseOptions) As VisualBasicParseOptions
+        Dim flag = Feature.ImplicitDefaultValueOnOptionalParameter.GetFeatureFlag
+        Return options.WithFeatures(options.Features.Concat(New KeyValuePair(Of String, String)() {New KeyValuePair(Of String, String)(flag, "true")}))
```

Comment:
That one should use it too (it probably wasn't changed because it was add in a child branch). Could you please change it as well?